src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -50,13 +50,22 @@
 							:close-after-click="true"
 							icon="icon-upload"
 							@click.prevent="clickImportInput">
-							{{ t('spreed', 'Upload new files') }}
+							{{ t('spreed', 'Upload from device') }}
 						</ActionButton>
 						<ActionButton v-if="canShareFiles"
 							:close-after-click="true"
 							icon="icon-folder"
 							@click.prevent="handleFileShare">
-							{{ t('spreed', 'Share from Files') }}
+							{{ t('spreed', 'Share from Nextcloud') }}
```

Comment:
There was a conversation in the design team when you were on vacation and we renamed everything across all clients

Reply:
Still doesn't work with white labeling.
If we want to change it we need to inject the theme name
```js
t('spreed', 'Share from {nextcloud}', {nextcloud: oc_defaults.productName})
```

Reply:
> Doesn't work with white labeling.
> Need to keep Files or we need to inject the the instance name from the Theming options and I'd like to avoid that.